src/main/java/seedu/address/logic/commands/tag/FilterByTagCommand.java

```diff
@@ -25,9 +24,9 @@ public class FilterByTagCommand extends Command {
 
             + " are tagged with the input tag.\n"
```

Comment:
I think it would be better to put 'tags' in plural form since there can be multiple inputs?

Reply:
Good catch! Have changed accordingly.